src/api/app/models/channel.rb

```diff
@@ -163,9 +163,9 @@ def add_channel_repos_to_project(tpkg, mode = nil)
 
     # defined in channel
     self.channel_targets.each do |ct|
-      next if mode == :skip_disabled and ct.disabled
+      next if mode == :skip_disabled && ct.disabled
       repo_name = ct.repository.extended_name
-      next unless mode==:enable_all or not ct.disabled
+      next unless mode==:enable_all || !ct.disabled
```

Comment:
I don't know if you agree or not in adding spaces... Improving readability using spaces is making it too hard? I don't see it... is just doing it more clear.